src/nidigital/metadata/functions_addon.py

```diff
@@ -142,3 +142,71 @@
         ],
     },
 }
+
+functions_additional_fetch_history_ram_cycle_information = {
+    'FancyFetchHistoryRAMCycleInformation': {
+        'python_name': 'fetch_history_ram_cycle_information',
+        'codegen_method': 'python-only',
+        'method_templates': [
+            {
+                'documentation_filename': 'default_method',
+                'method_python_name_suffix': '',
+                'session_filename': 'fancy_fetch_history_ram_cycle_information',
+            }
+        ],
+        'documentation': {
+            'description': '\nReturns the pattern information acquired for the specified cycles.\n\nIf the pattern is using the edge multiplier feature, cycle numbers represent tester cycles, each of which may\nconsist of multiple DUT cycles. When using pins with mixed edge multipliers, pins may return\nNIDIGITAL_VAL_PIN_STATE_NOT_ACQUIRED for DUT cycles where those pins do not have edges defined.\n'
```

Comment:
Does `PIN_STATE_NOT_ACQUIRED` exist as an enum value somewhere? It looks like this isn't correctly being turned into a link, and that would be one reason it wouldn't be.

Reply:
I'm working on another PR where I update to using enums everywhere. I initially thought of including this in that PR, but I don't really have to wait until that PR. I fixed it here.